src/Patient/ReportingFlows/TimeoutPrompts.js

```diff
@@ -0,0 +1,149 @@
+import React, { useState } from 'react';
+import BigConfirmationButton from '../Home/OneStepActions/BigConfirmationButton';
+import PhotoPrompt from '../../Components/Patient/PhotoPrompt';
+import { ThumbUp, Timer } from '@material-ui/icons';
+import { Box, Grow } from '@material-ui/core';
+import Colors from '../../Basics/Colors';
+import useStores from '../../Basics/UseStores';
+import FlatButton from '../../Components/FlatButton';
+import { makeStyles } from '@material-ui/core/styles';
+import { useTranslation } from 'react-i18next';
+import { t } from 'i18next';
+
+export default function ValidateTimePrompt() {
+  const [isTestReady, setIsTestReady] = useState(true);
+  const [isPendingValidation, setIsPendingValidation] = useState(true);
+  const { t } = useTranslation('translation');
+
+  const handleTimeElapsed = (validTest = true) => {
+    setIsTestReady(validTest);
+    setIsPendingValidation(false);
+  };
+
+  return (
+    <>
+      <Box height="250px">
+        {isPendingValidation ? (
+          <Box height={'80px'}>
+            <TransitionWrapper height="auto">
+              <h3>{t('patient.report.photo.help.wait')}:</h3>
+            </TransitionWrapper>
+          </Box>
+        ) : (
+          <Box height={'80px'} />
+        )}
+        <TransitionWrapper condition={isPendingValidation}>
+          <TimeElapsedButton handleClick={() => handleTimeElapsed(false)} />
+          <TimeElapsedButton valid handleClick={handleTimeElapsed} />
+        </TransitionWrapper>
+        {!isPendingValidation && <TestIsReady isTestReady={isTestReady} />}
+      </Box>
+    </>
+  );
+}
+
+const TimeElapsedButton = ({ handleClick, valid }) => {
+  return (
+    <Box padding="1em .5em">
+      <BigConfirmationButton
+        primaryColor={valid ? '#388E3C' : Colors.red}
+        bgColor={valid ? '#E8F5E9' : Colors.calendarRed}
+        icon={valid ? <ThumbUp /> : <Timer />}
```

Comment:
If you are using new colors here, just thow them in the Colors file to keep track of them